src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java

```diff
@@ -28,8 +28,11 @@ public static void main(String[] args) {
      * @return время в формате 'D H:m:s', где D - дни, H - часы, m - минуты, s - секунды
      */
     static String getPlayingTime(int playingTimeInSeconds) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        int D = playingTimeInSeconds / 86400;
+        int H = playingTimeInSeconds % 86400 / 3600;
```

Comment:
"Violation: 'nameVariable' was not used for variable naming
Solution: Rename variable"